docs/microphone.rst

```diff
@@ -0,0 +1,109 @@
+Microphone **V2**
+*****************
+
+.. py:module:: microbit.microphone
+
+This object lets you access the built-in microphone available on the
+micro:bit **V2**. It can be used to respond to sound. The microphone input
+is located on the front of the board alongside a microphone activity LED,
+which is lit when the microphone is in use.
+
+.. image:: microphone.png
+    :width: 300px
+    :align: center
+    :height: 240px
+    :alt: micro:bit with microphone LED on
+
+Sound events
+============
+The microphone can respond to a pre-defined set of sound events that are
+based on the amplitude and wavelength of the sound. The ``SoundEvent`` class
+is already instantiated and can be used with the ``microphone`` methods
+documented below.
+
+Functions
+=========
+
+.. py:function:: current_event()
+
+    * **return**: the name of the last recorded ``SoundEvent``,
+      ``SoundEvent.LOUD('loud')`` or ``SoundEvent.QUIET('quiet')``.
+
+.. py:function:: was_event(event)
+
+    * **event**: a sound event,  such as ``SoundEvent.LOUD`` or
+      ``SoundEvent.QUIET``.
+    * **return**: ``true`` if sound was heard at least once since the last
+      call, otherwise ``false``.
+
+.. py:function:: get_events():
```

Comment:
```suggestion
.. py:function:: get_events()
```

Not render correctly otherwise (doesn't have the microbit.microphone prefix):

![image](https://user-images.githubusercontent.com/29712657/100908217-29d4a180-34c3-11eb-9c6f-7d1d47e114d5.png)